tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -142,6 +145,38 @@ impl<OpOrder: PBSOrderMarker> CiphertextBase<OpOrder> {
         self.carry_modulus = other.carry_modulus;
         self._order_marker = other._order_marker;
     }
+
+    pub fn to_explicit<OtherOpOrder: PBSOrderMarker>(&self) -> &CiphertextBase<OtherOpOrder> {
+        self.try_to_explicit().expect("Wrong ciphertext type.")
+    }
```

Comment:
unsure on the naming here, maybe to_concrete_type ? or whatever the naming is when resolving a generic type to the actual type ? Maybe to_explicit is the right wording

With the implementation of fmt as proposed above you can unwrap on try explicit and get a nice error message

Reply:
Yes I've included better error handling.
About the naming: I'm not decided on anything. The most blatant use case is the `my_fun(ct.to_explicit())` call, which seemed clear enough, but `to_concrete_type` is fine as well. Since it's not technically a conversion we could just do `explicit` which will take less room in code

wyt ?

Reply:
I've changed to `to_concrete_type`